### PR TITLE
small readme updates

### PR DESCRIPTION
**Pull request by xelzi**
😎

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,28 +71,28 @@ export default withUtilityConnect(CreateCredentials, config);
...
+| `callbacks`   | `object` | Callback functions                         |                                      | No       | none     |
+| `uiTheme`     | `string` | UI color theme                             | `['light', 'dark']`                  | No       | 'light'  |
 
 #### scope
 
 Specifies the user flow. Defaults to `create`.
 
-- `create`: Opens the Utility Connect service in the "create" flow - input credentials will be used to create a user and corresponding utility credential record.
-- `update`: Opens the Utility Connect service in the "update" flow - input credentials will be used to update an existing utility credential record. Requires that you add `user.id` and `utilityCredential.id` in the `data` object.
+- `create`: Opens the Utility Connect service in the "create user" flow - input credentials will be used to create a user and corresponding utility credential record.
```
Comment (sdpeyton) [inline README.md:94]: should this be `create utility credentials` flow? We create both a user and utility credentials but the credentials feel like the primary focus

**Review by sdpeyton — APPROVED** [review]
thanks!

File: README.md [hunk 2]
```diff
@@ -114,27 +122,28 @@ The name used to reference the client organization.
...
+
+Data is expected in the following format for the `create` scope:
 
 ```javascript
 {
   user: {
     email: 'fake_email@example.com', // user's email
-    firstName: 'Falsey', // user's first name
-    lastName: 'Farcicle' // user's last name
+    firstName: 'First', // user's first name
```
Comment (sdpeyton) [inline README.md:133]: whyyyyyyyy 😭

File: README.md [hunk 3]
```diff
@@ -114,27 +122,28 @@ The name used to reference the client organization.
...
+    lastName: 'Name' // user's last name
```
Comment (sdpeyton) [inline README.md:134]: also if you are gonna change this, this should be `'Last`

Comment (xelzi) [reply README.md:134]: oops hahaha thats what I MEANT

File: README.md (hunk 1 above)
Comment (xelzi) [reply README.md:94]: hmmm lets leave for now but I think this is a good question to raise to the group when we try to align the docs

**Pull request merged**